### PR TITLE
fix: Improve export asset, reduce breakdown limit

### PR DESCRIPTION
**Pull request by webjunkie**
## Problem

Exports keep failing. Reasons: too many breakdowns, missing support for some result formats from trends...

## Changes

* reduce breakdown limit from 1000 to 500
  * it was making too huge CH queries
* remove auto-retry
  * mostly the export fails hard, no point in retrying
* remove sync checking with timeout
  * we don't probably want to keep requests open that long if not needed
  * one simple way makes less problems, less code
* ignore network errors in frontend polling
  * we don't want to say it failed when the server never told us so
* simplify error handling
* simplify url param handling
* rearrange some conditions for readability
* handle some concurrency issues
* add output for funnel "time to convert" results

## How did you test this code?

- checked tests and tried a bit

### CODE REVIEW

File: posthog/hogql_queries/insights/trends/test/__snapshots__/test_trends.ambr
Comment (mariusandra) [inline posthog/hogql_queries/insights/trends/test/__snapshots__/test_trends.ambr:1]: I've seen these cohorts flap all around. Not relevant for this PR, but we should fix this somehow...

File: posthog/tasks/exporter.py [hunk 1]
```diff
@@ -37,23 +39,21 @@
 
 # export_asset is used in chords/groups and so must not ignore its results
 @shared_task(
-    autoretry_for=(Exception,),
-    max_retries=5,
```
Comment (mariusandra) [inline posthog/tasks/exporter.py:41]: Wow, this might answer why exports sometimes take unbelievably long and still fail.

**Review by mariusandra — APPROVED** [review]
This all makes sense to me. Nice work!

File: posthog/tasks/exporter.py (hunk 1 above)
Comment (webjunkie) [reply posthog/tasks/exporter.py:41]: I actually think they have no clear "now it's failed" state... so this doesn't make them take longer, just try needlessly. To mark hard fails would be another todo, I'll see if there is time to do that.

**Pull request merged**